common/api/core-backend.api.md

```diff
@@ -4251,8 +4313,8 @@ export class RenderTimeline extends InformationRecordElement {
     protected constructor(props: RenderTimelineProps, iModel: IModelDb);
     // @internal (undocumented)
     static get className(): string;
-    // @alpha (undocumented)
-    protected collectReferenceIds(ids: Id64Set): void;
```

Comment:
what is the policy for this kind of change? `collectReferenceIds` still exists as alpha, inherited through `Element` which has it as `@alpha` (while further `Entity` has it as `@internal`). Am I supposed to prevent api-extractor from seeing this for docs reasons or can we leave be?

Reply:
I reintroduced an override with the original tag